code/web/src/modules/product/Detail.js

```diff
@@ -42,6 +42,7 @@ class Detail extends PureComponent {
   }
 
   render() {
+    // console.log(this.state)
```

Comment:
I thought you deleted this console.log before pushing but maybe I misunderstood. Want to delete before merging?

Reply:
Good catch, I missed that one. I've removed it.